Develop the mint sign up page clone

sonegillis opened the pull request:
1. Create the html structure
2. Implement the navbar, form and footer in the body
3. Style the navbar
4. Style the form
5. Style the footer

sonegillis committed:
Create necessary files

sonegillis committed:
Structure the index.html

sonegillis committed:
Merge pull request #1 from sonegillis/build-html-struct

Structure the index.html

sonegillis committed:
Create the three main sections

sonegillis committed:
Merge pull request #2 from sonegillis/create-sections

Create the three main sections

sonegillis committed:
FILL FORM ELEMENTS
1. Add the labels and input fields
2. Add the form headers and footers

sonegillis committed:
Merge pull request #3 from sonegillis/fill-form-section

FILL FORM ELEMENTS

wathigo committed:
Navbar structure implemented

wathigo committed:
Merge pull request #4 from sonegillis/implement-navbar-structure

Navbar structure implemented

sonegillis committed:
Create .stickler.yml

wathigo committed:
Footer structure implemented

wathigo committed:
stickler.yml file modified

wathigo committed:
Merge pull request #5 from sonegillis/implement-footer-structure

Footer structure implemented

wathigo committed:
Add the navbar structure

wathigo committed:
Navbar Styling Implemented
1. Float the elements to the left.
2. Add a blue colour to the button.
3. Give a height to the navbar.

wathigo committed:
Css Properties Modification
1. Modified the button background-color property.
2. Modified the link color property.

wathigo committed:
Added the box-sizing property

wathigo committed:
Css Properties Modified
1. Removed the box-sizing property.
2. Modified the padding property.

wathigo committed:
Css Properties Modified
1. Width property of the button element removed.
2. Modified the padding property of the button element.

wathigo committed:
Merge pull request #6 from sonegillis/style-navbar

Style navbar

sonegillis committed:
style the sign up form

sonegillis committed:
Merge pull request #7 from sonegillis/style-form

style the sign up form

wathigo committed:
Styling of the footer element implemented

wathigo committed:
Merge pull request #8 from sonegillis/style-footer

Styling of the footer element implemented

wathigo committed:
Update README.md

sonegillis committed:
Update README.md

JamesOkunlade reviewed (changes requested):
### Hello guys!! 

Nice work with the HTML forms project. However, there are some avoidable errors in your code. Parse your HTML code through the [W3C markup validator](https://validator.w3.org/) to detect and fix these errors.

Also, you have not completely paid attention to details. I don't see the "Intuit" logo and icons on the top of your form. Please implement them. 

You should work on the colors of your buttons and the width of your form as well to look more like the main site. 

When you're done request for another code review and me as the TSE so I can follow up on it.

Happy coding!

wathigo committed:
Removed attributes with empty values

wathigo committed:
Merge pull request #10 from sonegillis/fix-errors

Removed attributes with empty values

wathigo committed:
Chagens Made
1. Implemented a logo.
2. Implemented icons.
3. Modified the width of the form.

sonegillis committed:
Merge pull request #11 from sonegillis/implement-icons

Changes Made

sonegillis committed:
Modify button colors

sonegillis committed:
Changed background color of body

sonegillis committed:
Merge pull request #12 from sonegillis/modify-buttons

Modify buttons

sonegillis committed:
fix syntax errors in html

sonegillis committed:
Merge pull request #13 from sonegillis/fix-errors

fix syntax errors in html

sonegillis committed:
fix more errors

sonegillis committed:
Merge pull request #14 from sonegillis/fix-errors

fix more errors

sonegillis committed:
Correct lint errors

sonegillis committed:
Merge pull request #15 from sonegillis/correct-linter-errors

Correct lint errors

sonegillis committed:
Correct more litt errors

sonegillis committed:
Merge pull request #16 from sonegillis/correct-linter-errors

Correct more litt errors

JamesOkunlade commented on css/styles.css:
```diff
@@ -0,0 +1,211 @@
+fieldset {
+  border: 0;
+  margin-left: auto;
+  margin-right: auto;
```
Refactor line 69 and 70 into one line margin: 0 auto;

JamesOkunlade reviewed (approved):
### Great improvement!!

This looks good to me now. Looks good across multiple browsers too.

Please merge develop to master, close pull request and fill the project completion form.

Good luck with your future projects. :+1::+1:

sonegillis committed:
Update styles.css
